Aspire Budgeting/Views/CategoryDetailsView.swift

```diff
@@ -84,8 +126,9 @@ struct CardExpandedView_Previews: PreviewProvider {
         .CardDetails(title: title,
                      bannerGradient: bannerGradient,
                      budgetedTotal: AspireNumber(stringValue: "$500"),
-                     spentTotal: AspireNumber(stringValue: "$30"),
-                     availableTotal: AspireNumber(stringValue: "40"))
+                     spentTotal: AspireNumber(stringValue: "-$30"),
+                     availableTotal: AspireNumber(stringValue: "40"),
+                     categories: MockProvider.cardViewItems[0].categories, tintColor: .blueFondEndColor)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/072125a9dd8d39bfc48236c2af080b4a5a19a520/Aspire Budgeting/Views/CategoryDetailsView.swift#L131"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>CategoryDetailsView.swift:131</code></td>
    </tr>
  </tbody>
</table>